W-15837181 clarify doc types (#33)
mulesoft/docs-idp: modules/ROOT/pages/document-processing.adoc

include::partial$document-action.adoc[tag=definition]

> IDP provides preconfigured Invoice and Purchase Order templates, which you can select when creating custom document actions. Each template provides a set of expected fields to extract from the document and generate a structured response. When creating a new document action, choose the type that better aligns with the document to process. For example, the Invoice type can process invoices and similar documents such as receipts. If none of these types align with your documents, use the Generic type to fully customize the fields to extract by asking questions in natural language. 

Reviewer: ```suggestion
IDP provides preconfigured *Invoice* and *Purchase Order* templates for creating custom document actions. Each template provides a set of expected fields to extract from the document into a structured response. When creating a new document action, choose the type that aligns best with the document to process. For example, the *Invoice* type can process invoices, receipts, and similar documents. If none of these types align with your documents, use the *Generic* type to fully customize the fields to extract by asking questions in natural language. 
```

Reviewer: *bold* only if UI text, can these be lower case, too?
